wallet/src/de/schildbach/wallet/ui/WalletTransactionsFragment.java

```diff
@@ -317,10 +272,8 @@ private void showEmptyView() {
         recyclerView.setVisibility(View.INVISIBLE);
     }
 
-    @Override
-    public void onLoaderReset(final Loader<List<Transaction>> loader) {
-        // don't clear the adapter, because it will confuse users
-    }
+    //TODO: Do we need to handle this?
```

Comment:
Most of the below code should be handled by LiveData/ViewModel, however `walletChangeReceiver` part can be ignored since we do not use ACTION_WALLET_REFERENCE_CHANGED anymore (it is a legacy code).

Reply:
As agreed with @tomasz-ludek , throttling will be implemented in https://github.com/dashevo/dash-wallet/pull/526.